teacher_test.rb

```diff
@@ -0,0 +1,62 @@
+require_relative 'person'
```

Comment:
- [ ] All your tests are not passed for example classroom, teacher tests which are required for this project are not passed 😢  as you can see on the screenshot ⬇️. Please make sure that all tests are well implemented and passed as required for this project 👍

Reply:
@codecaiine I kindly request that you provide us with a screenshot to verify the comment you have made. However, I would like to clarify that I have a different understanding of the situation, as I have a screenshot that indicates the successful passing of the teacher and the classroom class test. The attached images display the results of the test I recently conducted in the said classes.

**This is for the Teacher Test**
![teacher-test-again](https://user-images.githubusercontent.com/32777408/225934656-46869795-191f-4518-936d-3a795abefe1b.png)

**This is for the Classroom Test**
![classroom-test-again](https://user-images.githubusercontent.com/32777408/225935050-9868e455-ae05-4802-9e6b-427e3ba1a53e.png)